support-frontend/assets/components/headers/header/header.tsx

```diff
@@ -13,6 +14,7 @@ export type PropTypes = {
 	utility?: JSX.Element;
 	countryGroupId: CountryGroupId;
 	display?: 'navigation' | 'checkout' | 'guardianLogo' | void;
+	participations?: Participations;
```

Comment:
Because this is optional, not every use of the header is directing to the `/delivery` version of the url (eg error pages). Maybe I should go back and add this in but I focused on the other landing pages for now